app/script/view_model/content/InputBarViewModel.js

```diff
@@ -314,9 +320,16 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
 
   _show_upload_warning(image) {
     const isGif = image.type === 'image/gif';
-    const stringId = isGif ? z.string.modalGifTooLargeMessage : z.string.modalPictureTooLargeMessage;
+    const messageStringId = isGif ? z.string.modalGifTooLargeMessage : z.string.modalPictureTooLargeMessage;
+    const titleStringId = isGif ? z.string.modalGifTooLargeHeadline : z.string.modalPictureTooLargeHeadline;
+
+    const modalOptions = {
+      text: {
+        message: z.l10n.text(messageStringId, z.config.MAXIMUM_IMAGE_FILE_SIZE / 1024 / 1024),
```

Comment:
Let's put the `1024` in a const inside the config... it's used widely across the project.

Reply:
```
const BYTES_IN_KB = 1024;
const BYTES_IN_MB = BYTES_IN_KB * BYTES_IN_KB;
```